src/components/MessagesList/MessagesGroup/Message/MessagePart/FilePreview.vue

```diff
@@ -375,18 +375,14 @@ export default {
 		},
 
 		isViewerAvailable() {
-			if (!OCA.Viewer) {
+			if (!OCA.Viewer || !OCA.Viewer.availableHandlers?.length) {
 				return false
 			}
 
-			const availableHandlers = OCA.Viewer.availableHandlers
-			for (let i = 0; i < availableHandlers.length; i++) {
-				if (availableHandlers[i]?.mimes?.includes && availableHandlers[i].mimes.includes(this.file.mimetype)) {
-					return true
-				}
-			}
-
-			return false
+			return OCA.Viewer.availableHandlers.some(handler => {
+				return handler?.mimes?.includes?.(this.file.mimetype)
+					|| handler?.mimes?.includes?.(handler?.mimesAliases?.[this.file.mimetype])
+			})
```

Comment:
```suggestion
			return OCA.Viewer.mimetypes?.includes?.(this.file.mimetype)
```

Reply:
Server (Files) uses similar approach, but for comparison: `OCA.Viewer?.mimetypesCompare?.includes(this.fileInfo.mimetype)`

Reply:
compare is for split-view support